governance.md

```diff
@@ -1,34 +1,61 @@
 ## Overview
 
-This document defines the maintenance policy for the [Kyma](../../../) organization. All repositories in this organization must follow all the rules and guidelines to assure quality and consistency.
+This document defines the ownership policy and the decision-making process within the [Kyma](../../../) organization.
 
-The Kyma project also includes the [Kyma Incubator](https://github.com/kyma-incubator) organization. It is a place where all new projects start in a more relaxed environment that facilitates their rapid growth. At that stage, they do not have to comply with all the rules that govern the Kyma organization. Once the incubating project is ready to become a part of the main Kyma organization, you must adjust it to all the standards.
+## Scope
 
-## Maintenance policy
+All repositories in the Kyma organization must follow the official guidelines, contributing rules, and the governance process to assure quality and consistency.
 
-The Kyma maintainers are a group of people that have special privileges in the repositories of the [GitHub](../../../) organization. The maintainers have the ability to approve pull requests after a detailed review of the code and its functionality. The rules for contributing to Kyma are available in the [`CONTRIBUTING.md`](CONTRIBUTING.md) document.
+The Kyma project also includes the [Kyma Incubator](https://github.com/kyma-incubator) organization. It is a place where all new projects start in a more relaxed environment that facilitates their rapid growth. At that stage, they do not have to comply with all the rules that govern the Kyma organization. Once the incubating project is ready to become a part of the main Kyma organization, adjust it to all the standards.
 
-## Maintainers' responsibilities
+## Ownership policy
 
-With great power comes great responsibility. Maintainers not only make changes and approve pull requests but also truly care about their projects.
+Kyma repositories are owned by code owners who are a group of people with special privileges in the repositories of the [GitHub](../../../) organization. Each repository has a separate `CODEOWNERS` file located at its root. The file specifies persons who have the ability to approve contribution to the part of the repository they own. after a detailed review of the related pull requests. Although the name suggests only the code ownership, the `CODEOWNERS` file is not only about the code but the content in general. Apart from the developers, you can define any relevant parties as code owners. For example, technical writers are set up as the owners of all `.md` documents in the Kyma repositories and SIG/WG members are the owners of their SIG/WG's folders' content.
```

Comment:
`The file specifies persons who have the ability to approve contribution to the part of the repository they own, after a detailed review of the related pull requests. ` -> changed dot in the middle of the sentence into a comma